src/main/java/task/ITaskController.java

```diff
@@ -0,0 +1,39 @@
+package task;
+
+import java.util.ArrayList;
+
+public interface ITaskController {
+	/**
+	 * Method call for obtaining ArrayList of Task user has.
+	 * @return ArrayList<Task>
+	 * @throws EmptyTaskListException
+	 */
+	public ArrayList<Task> getTasks() throws EmptyTaskListException;
+	/**
+	 * Adds new task to controller. Task will be stored in ArrayList.
+	 * @param description task specified by user
+	 * @return Task object
+	 */
+	public String addTask(Task newTask);
+	/**
+     * Marks task based on task index specified
+     * @param taskIndex
+     * @return String message displayed by Duke for user
+     * @throws TaskIndexOutOfRangeException
+     * @throws TaskMarkException
+     */
```

Comment:
It might be better if the indexes are the same.

Reply:
Yes, I should fix the indent here